pacehrh/R/pace_coverage_config.R

```diff
@@ -0,0 +1,72 @@
+#' Load CoverageRates Values
```

Comment:
@RHull-IDM the build broke with this file: could you investigate what's wrong? thanks!
```
pace_coverage_config.R
ERROR: unable to collate and parse R files for package 'pacehrh'
```

Reply:
Looking into it